### ISSUE TITLE
[css-grid] `grid-template` without rows defined outputs empty rows template

### ISSUE THREAD

**Issue by Dan503** [postcss/autoprefixer#1049]
This source code:
````css
.grid {
  display: grid;
  grid-gap: 10px;
  grid-template:
  "a  b"      /* <<< note that rows are not explicitly set */
  "c  d" /
  1fr 1fr;
}

.cell-A {
  grid-area: a;
}
...
````
Compiles to
````css
.grid {
  display: -ms-grid;
  display: grid;
  grid-gap: 10px;
  -ms-grid-rows: ; /* <<<<  Note the lack of rows */
  -ms-grid-columns: 1fr 10px 1fr;
      grid-template: "a b"
 "c d"/
 1fr 1fr;
}

.cell-A {
  -ms-grid-row: 1;
  -ms-grid-column: 1;
  grid-area: a;
}
...
````
Expected output:
````css
.grid {
  display: -ms-grid;
  display: grid;
  grid-gap: 10px;
  -ms-grid-rows: auto 10px auto; /* <<<<  rows filled in with `auto` values */
  -ms-grid-columns: 1fr 10px 1fr;
      grid-template: "a b"
 "c d"/
 1fr 1fr;
}

.cell-A {
  -ms-grid-row: 1;
  -ms-grid-column: 1;
  grid-area: a;
}
...
````

**Comment by yepninja** [comment]
Thank you for the good examples. They will be very suitable for tests.

**Comment by Dan503** [comment]
@yepninja this is a high priority issue that should get fixed before the article is published.

**Comment by yepninja** [comment]
Released in 8.5.2.
